cypress/integration/unit_tests/manage_disaster_test.js

```diff
@@ -314,6 +315,35 @@ describe('Unit tests for manage_disaster.js', () => {
                 ' and must specify either damage asset or map bounds');
   });
 
+  it('nonexistent asset not ok', () => {
+    const missingSnapPath = 'whereisasset';
+    setUpAssetValidationTests()
+        .then(() => {
+          const data = createDisasterData(['NY']);
+          data.asset_data.damage_asset_path = 'pathnotfound';
+          data.asset_data.snap_data.paths.NY = missingSnapPath;
+          disasterData.set(getDisaster(), data);
+          Promise.all([
```

Comment:
do you not need a return before this Promise.all?

Reply:
Great point. But actually these things don't return Promises anyway, so we can just execute them :)